source/blog/2022-05-19-announcing-hanami-200alpha8.html.markdown

```diff
@@ -0,0 +1,90 @@
+---
+title: Announcing Hanami v2.0.0.alpha8
+date: 2022-05-19 12:00:00 UTC
+tags: announcements
+author: Tim Riley
+image: true
+excerpt: >
+  New base action and view classes, and a few fixes
+---
+
+After a month’s break, we’re back with another Hanami 2.0.0 alpha release for you all!
+
+This release includes new base action and view classes, plus a few small fixes.
+
+## New base action and view classes
+
+This release includes new base classes for actions and views that integrate with their surrounding Hanami application: `Hanami::Application::Action`, `Hanami::Application::View`, and `Hanami::Application::View::Context`. Your base classes should now look like this:
+
+```ruby
+require "hanami/application/view"
```

Comment:
What do you think about file names to better explain the structure?

Reply:
I've put a filename comment at the top of each code snippet now 😄